recipes/stockwell/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "stockwell" %}
+{% set version = "1.0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stockwell-{{ version }}.tar.gz
+  sha256: 885fb85c40063f00336755afa178a0445f5520015c6bfb1791ab346220ede399
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy
```

Comment:
See the following about building against `numpy`:
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy